Documentation/Elastics/ElasticSystem.md

```diff
@@ -0,0 +1,76 @@
+# Elastic system (experimental)
```

Comment:
For consistency maybe we don't need the (experimental) here? Some other experimental features like dialog and hand coach do not have that on their pages.

Reply:
I think that having this listed as experimental is good. We have other ones that don't have that but should, and this has led to some confusion about the expected quality level of certain subcomponents.